src/App.js

```diff
@@ -14,11 +14,70 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    values: [],
+  }
+
+  render() {
+    const { values } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          { values.length > 0
+            ? (
+              <>
+                Selected good:
+                {' '}
+                {values}
+                <button
+                  type="button"
+                  onClick={() => {
+                    this.setState({ values: [] });
+                  }}
+                >
+                  x
+                </button>
+              </>
+            )
+            : 'No goods selected'
+          }
+        </h1>
+        <ul>
+          {' '}
+          { goodsFromServer.map(item => (
+            <li key={item} className="list__item">
+              <span className="list__text">{item}</span>
+              {values.includes(item)
+                ? (
+                  <button
+                    type="button"
+                    className={values === item ? 'active' : ''}
```

Comment:
For switching classes, it's better to use 'classnames' library (not ternary operator with empty strings)
Not obligatory for this task, but use it in the future